script/deploy/holesky/Deploy_Test_RewardsCoordinator.s.sol

```diff
@@ -91,4 +91,39 @@ contract Deploy_Test_RewardsCoordinator is ExistingDeploymentParser {
             address(rewardsCoordinatorImplementation)
         );
     }
+
+    function _deployImplementation() internal {
+        // Existing values for current RewardsCoordinator implementationt on holesky
+        require(
+            REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS == 604800,
+            "REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS must be 604800"
+        );
+        require(
+            REWARDS_COORDINATOR_MAX_REWARDS_DURATION == 6048000,
+            "REWARDS_COORDINATOR_MAX_REWARDS_DURATION must be 31536000"
```

Comment:
incorrect require?

Reply:
https://holesky.etherscan.io/address/0xAcc1fb458a1317E886dB376Fc8141540537E68fE#readProxyContract
The value check is correct. The comment needs to be updated.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/692/commits/d72bbfa519476274795e64b7fd58f6418e81e687
Updated